ion-schema/src/result/invalid_schema_error.rs

```diff
@@ -0,0 +1,165 @@
+use crate::result::{HasIslSourceLocation, IonSchemaError, IonSchemaResult, IslSourceLocation};
+use std::collections::BTreeSet;
+use std::fmt::{Display, Formatter, Write};
+use thiserror::Error;
+
+/// Collects multiple errors to report them together.
+///
+/// ```ignore
```

Comment:
❗ This is `ignore` because doc tests can only use things that are visible outside of the crate, and `InvalidSchemaErrorCollector` is only `pub(crate)`.